docs/src/main/paradox/release-notes/10.4.x.md

```diff
@@ -0,0 +1,20 @@
+# 10.4.x release notes
+
+## 10.4.0
+
+This release contains one small improvement and some dependency updates. It should be a drop in replacement
+for Akka HTTP 10.2.x.
```

Comment:
should we also mention the license change here? something similar to https://github.com/akka/akka/blob/main/akka-docs/src/main/paradox/project/migration-guide-2.6.x-2.7.x.md#migration-guide-26x-to-27x

Reply:
we should probably also mention that Akka 2.7.0 is required, dropping support for 2.6. and 2.5

Reply:
I was back and forth, wrote something about both those and then removed. I'll re-add.

Reply:
Trying to address in [ecafb91](https://github.com/akka/akka-http/pull/4171/commits/ecafb912617a35bbc4927bc0dfc85921ffe1eb77)